### PR TITLE
Update to version 1.0

### LINKED ISSUE #1: Version 1.0 is not supported
The translator needs to be updated to support version 1.0 of the schema.

### PR DESCRIPTION
**Pull request by jasondegraw**
Fixes #1. There are a lot of changes here, but most of the changes are related to the use of lxml (again). Mostly having to do with namespaces (again). The changes that are not related to lxml are

 * Use the `BuildingSync` tag at root
 * Fix a couple of containerized lists

There may be more to change, but I'll go ahead and make the PR now. Should be merged after #10.

### CODE REVIEW

File: read211_tests.py [hunk 1]
```diff
@@ -61,12 +65,15 @@ def validate(filename, schema, instance):
...
+    def setUp(self):
+        self.maxDiff = None
+
     def test_main_style_call(self):
         for file in test_files:
             warnings.simplefilter("ignore")
             wb = loadxl.load_workbook(file)
             warnings.simplefilter("default")
-            std211 = read211.read_std211_xls(wb)
+            std211 = read211.read_std211_xlsx(wb)
```
Comment (nllong) [inline read211_tests.py:76]: does this mean it won't work with xls files?

**Review by nllong — CHANGES_REQUESTED** [review]
Can you make the resulting XML file be pretty printed?

```
<?xml version="1.0" encoding="UTF-8"?><BuildingSync xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://buildingsync.net/schemas/bedes-auc/2019" xsi:schemaLocation="http://buildingsync.net/schemas/bedes-auc/2019 ../BuildingSync.xsd"><Facilities><Facility><Sites><Site><Address><StreetAddressDetail><Simplified><StreetAddress>9999 Tulie Circle</StreetAddress></Simplified></StreetAddressDetail><City>Atlanta</City><State>GA</State><PostalCode>99999</PostalCode><PostalCodePlus4>99999-9999</PostalCodePlus4></Address><Buildings><Building ID="Building"><PremisesName>Test Building Input Data</PremisesName><BuildingAutomationSystem>true</BuildingAutomationSystem><HistoricalLandmark>false</HistoricalLandmark><YearOfConstruction>1888</YearOfConstruction><YearOfLastEnergyAudit>2001</YearOfLastEnergyAudit><YearOfLastMajorRemodel>2001</YearOfLastMajorRemodel><Subsections><Subsection><FootprintShape>Rectangular</FootprintShape><Sides><Side><WallID IDref="Wall1"/><WindowID IDref="Fenestration1"><WindowToWallRatio>0.25</WindowToWallRatio></WindowID></Side></Sides><RoofID IDref="Roof1"/>
```

to

```
<?xml version="1.0" encoding="UTF-8"?><BuildingSync xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://buildingsync.net/schemas/bedes-auc/2019" xsi:schemaLocation="http://buildingsync.net/schemas/bedes-auc/2019 ../BuildingSync.xsd">
	<Facilities>
		<Facility>
			<Sites>
				<Site>
					<Address>
						<StreetAddressDetail>
							<Simplified>
								<StreetAddress>9999 Tulie Circle</StreetAddress>
							</Simplified>
						</StreetAddressDetail>
						<City>Atlanta</City>
						<State>GA</State>
						<PostalCode>99999</PostalCode>
						<PostalCodePlus4>99999-9999</PostalCodePlus4>
					</Address>
					<Buildings>
					<Building ID="Building">
...
```

Also, it looks like there are still a bunch of UDFs. Am I correct to expect these to be minimized after our call with Jim?

File: read211_tests.py (hunk 1 above)
Comment (jasondegraw) [reply read211_tests.py:76]: @nllong As far as I know, yes. There's an older python module that will apparently read xls files, but I don't believe that openpyxl will read those. And the code to extract all of the checkboxes is certainly xlsx only. If there's a need to read those files, there isn't a great solution other than converting the files to the newer xlsx format.

**Comment by jasondegraw** [comment]
@nllong If you're using the translator as a script, hopefully the `-p` (or `--pretty`) option should get you a pretty printed xml. It works for me on Windows, could you give that a try?

On UDFs, yes, this just gets us up to 1.0. There are some easy resolutions that can go in once this is merged, but a lot will need further discussion.

**Comment by nllong** [comment]
gotcha, the `-p` worked. 

Oh, can you fix the CLI help text to remove BSXML?

```
nlong:std211-translator (update-to-v1 %=) [3.6.5-std211] [2.8.0.a7a1f79e98]$ python read211.py --help
usage: read211.py [-h] [-p] [-o OUTFILE] [-g] [-v] INFILE

Convert ASHRAE Std. 211 Workbook into BSXML (use BuildingSync or BSync).

positional arguments:
  INFILE

optional arguments:
  -h, --help            show this help message and exit
  -p, --pretty          output pretty xml
  -o OUTFILE, --output OUTFILE
                        file to save BSXML output in
  -g, --groupspaces     group spaces into zones by principal HVAC type
  -v, --verbose         operate verbosely
```

**Comment by jasondegraw** [comment]
@nllong I caught those when I generated the documentation, so I'm pretty sure the only BSXMLs left after #13 are variable names. Which we can also get rid of if you prefer.

**Review by nllong — APPROVED** [review]

**Pull request merged**